miningthetext.py

```diff
@@ -0,0 +1,16 @@
+'''Gracey Wilson
+Software Design Spring 2017
+Mini Project 3: Text Mining
+
+A script that grabs the text from ProjectGutenberg online and pickles it'''
+
+import requests
+romeo_juliet_full_text = requests.get('http://www.gutenberg.org/cache/epub/1112/pg1112.txt').text
+print(romeo_juliet_full_text)
+
+import pickle
```

Comment:
By convention, we usually put all import statements at the top of the file (but after header comment). In this case, I recommend putting this import statement after `import requests`. This is exceptionally short python script so this misplaced import statement does not matter much, but when dealing with substantially longer python file, putting all import statements at the top allows readers to have a mental picture of what kind of external programs/modules/libraries are used in the file.